packages/bbui/src/Form/Core/File.svelte

```diff
@@ -57,7 +58,16 @@
 <!-- svelte-ignore a11y-no-static-element-interactions -->
 <!-- svelte-ignore a11y-click-events-have-key-events -->
 <div class="field">
-  {#if value}
+  {#if statusText}
```

Comment:
Needed a state for the File component where it didn't take a `File` type, just some text. And I thought this was cleaner than mixing types on the `value` prop.